frontend/wallet/src/render/components/Downloader.re

```diff
@@ -1,5 +1,5 @@
 [@bs.val] [@bs.scope "window"]
-external downloadKey:
+external downloadCoda:
   (string, (int, int) => unit, Belt.Result.t(unit, string) => unit) => unit =
   "";
```

Comment:
I could be missing something but it doesn't look like this matches the function in the preload

Reply:
I think I messed up the initial preload definition, should be fixed now.